src/System.Data.DataSetExtensions/Directory.Build.props

```diff
@@ -1,6 +1,8 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
+    <!-- AssemblyVersion is frozen to that which originally shipped in 2.1 -->
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
Instead, let's just stop building the `netstandard2.0` ref.  We aren't building a netstandard (or netfx) lib any more, nor are we building the package.  Let's let the netcoreapp version move on.